yarn.lock

```diff
@@ -10262,178 +11323,6 @@ terriajs-tiff-imagery-provider@2.13.3:
   dependencies:
     geotiff "^2.0.7"
 
-terriajs@8.7.10:
```

Comment:
This looks wrong, there's no update to package.json in this PR but yarn.lock is changed a lot, and some essential packages are removed from yarn.lock?

Reply:
I have reverted yarn.lock for now - the diff you see is from changes to package.json in terriajs (current main and cjs-to-esm).